bokeh/server/session.py

```diff
@@ -61,10 +68,38 @@ def connection_count(self):
     def seconds_since_last_unsubscribe(self):
         return current_time() - self._last_unsubscribe_time
 
+    def add_periodic_callback(self, callback):
```

Comment:
as with ClientSession maybe this method should be private `_add_periodic_callback`